tests/unit/accordion/accordion_methods.js

```diff
@@ -7,10 +7,17 @@ var equalHeight = TestHelpers.accordion.equalHeight,
 module( "accordion: methods", setupTeardown() );
 
 test( "destroy", function() {
-	expect( 1 );
+	expect( 2 );
 	domEqual( "#list1", function() {
 		$( "#list1" ).accordion().accordion( "destroy" );
 	});
+
+	// Animate
+	domEqual( "#list1", function() {
+		$( "#list1" ).accordion()
+			.accordion( "option", "active", 1 )
+			.accordion( "destroy" );
+	});
```

Comment:
Im happy to keep or remove them i'm not really sure either but here is the original conversation where @jzaefferer said to leave them https://github.com/jquery/jquery-ui/pull/1369/files#r19285585